src/scenekit.cs

```diff
@@ -1766,6 +1890,248 @@ interface SCNNode : SCNAnimatable, SCNBoundingVolume, SCNActionable, NSCopying,
 		[Export ("nodeWithMDLObject:")]
 		SCNNode FromModelObject (MDLObject mdlObject);
 #endif
+
+		// From SCNNode (Transforms) Category
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Static]
+		[Export ("localUp")]
+		SCNVector3 LocalUp { get; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Static]
+		[Export ("localRight")]
+		SCNVector3 LocalRight { get; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Static]
+		[Export ("localFront")]
+		SCNVector3 LocalFront { get; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("worldUp")]
+		SCNVector3 WorldUp { get; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("worldRight")]
+		SCNVector3 WorldRight { get; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("worldFront")]
+		SCNVector3 WorldFront { get; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("lookAt:")]
+		void Look (SCNVector3 worldTarget);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("lookAt:up:localFront:")]
+		void Look (SCNVector3 worldTarget, SCNVector3 worldUp, SCNVector3 localFront);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("localTranslateBy:")]
+		void LocalTranslate (SCNVector3 translation);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("localRotateBy:")]
+		void LocalRotate (SCNQuaternion rotation);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("rotateBy:aroundTarget:")]
+		void Rotate (SCNQuaternion worldRotation, SCNVector3 worldTarget);
+
+		// From SCNNode (SIMD) Category
```

Comment:
I don't think we need the SIMD versions of API already provided using other types (and I don't think these API behave differently, they just accept different types). Speed-wise there's nothing to gain (in fact our SIMD versions would probably be slower, since we have to do additional conversions under the hood).

I would just comment all these out, explaining why we don't need them.

OTOH it might be useful to have (explicit) conversion operators between the SCN* and N* types, to make it easier to use the SCN API with N* types.

Reply:
Yes, I agree on the slower part (unsure if really noticeable) but I think we should have the API since Apple provides it.

So I see 3 options here:

1. Create the explicit conversion operators between the SCN* and N* types and expose the same Simd API but using a managed cast.
2. Leave the API as is, calling native API
3. Commenting it out for now and re-enable in the future if someone asks for it.

Since Xcode 9.1 release is coming really soon I might lean towards option 3 + be filling a bug about explicit conversion operators between the SCN* and N* types so we can have the other SceneKit API in.

@spouliot thoughts?